org.freedesktop.weston.yaml

```diff
@@ -0,0 +1,98 @@
+app-id: org.freedesktop.weston
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: weston
+rename-desktop-file: weston.desktop
+rename-icon: wayland
+finish-args:
+- --socket=wayland
+- --device=dri
+- --filesystem=/run/user/
+
+modules:
+- name: mtdev
+  sources:
+  - type: git
+    url: http://bitmath.org/git/mtdev.git
+    tag: "v1.1.6"
+    # fatal: dumb http transport does not support shallow capabilities
+    disable-shallow-clone: true
+  buildsystem: autotools
+
+- name: check
+  sources:
+  - type: git
+    url: https://github.com/libcheck/check.git
+    tag: "0.15.2"
+  buildsystem: cmake-ninja
+  builddir: true
+
+- name: libevdev
+  sources:
+  - type: git
+    url: https://gitlab.freedesktop.org/libevdev/libevdev.git
+    tag: "libevdev-1.12.0"
+  buildsystem: meson
+  builddir: true
+
+- name: libinput
+  sources:
+  - type: git
+    url: https://gitlab.freedesktop.org/libinput/libinput.git
+    tag: "1.19.3"
+  buildsystem: meson
+  builddir: true
+  config-opts:
+  - -Dlibwacom=false
+  - -Ddebug-gui=false
+  - -Dtests=false
+
+- name: seat
+  sources:
+  - type: git
+    url: https://git.sr.ht/~kennylevinsen/seatd
+    tag: "0.6.3"
+  buildsystem: meson
+  builddir: true
+
+- name: weston
+  sources:
+  - type: git
+    url: https://gitlab.freedesktop.org/wayland/weston.git
+    tag: "10.0.0"
+  buildsystem: meson
+  builddir: true
+  config-opts:
+  - -Dbackend-rdp=false
+  - -Dweston-launch=false
+  - -Dxwayland=false
+  - -Dsystemd=false
+  - -Dpipewire=false
+  - -Dcolor-management-colord=false
+  - -Dlauncher-logind=false
+  - -Dlauncher-libseat=true
+  - -Dshell-fullscreen=false
+  - -Dshell-ivi=false
+  - -Dshell-kiosk=false
+
+- name: launcher
```

Comment:
this module should be merged with the one above. Use `post-install` for the install steps.